### PR TITLE
I think this is a better background color!

### PR DESCRIPTION
**Pull request by amirmishani**

### CODE REVIEW

**Comment by amirmishani** [comment]
Well let's get Aaron to look at this first.

File: jumbotron.css [hunk 1]
```diff
@@ -2,4 +2,5 @@
 body {
   padding-top: 50px;
   padding-bottom: 20px;
+  background: lavender;
```
Comment (amirmishani) [inline jumbotron.css]: Hey this is good! I like it.

Comment (amirmishani) [inline jumbotron.css:5]: I like this is. good work!

**Comment by ghost** [comment]
Brilliant work! :D

**Pull request merged**